src/components/activityEventElems/ActivityElement/ActivityElement.tsx

```diff
@@ -0,0 +1,122 @@
+import { Trans } from '@lingui/macro'
+import EtherscanLink from 'components/EtherscanLink'
+import FormattedAddress from 'components/FormattedAddress'
+import { ProjectVersionBadge } from 'components/ProjectVersionBadge'
+import { ThemeContext } from 'contexts/themeContext'
+import { useV2V3TerminalVersion } from 'hooks/V2V3TerminalVersion'
+import { useContext } from 'react'
+import { formatHistoricalDate } from 'utils/format/formatDate'
+import { contentLineHeight, smallHeaderStyle } from '../styles'
+import { ActivityElementEvent } from './activityElementEvent'
+
+const DetailsContainer: React.FC = ({ children }) => {
+  return (
+    <div
+      style={{
+        display: 'flex',
+        justifyContent: 'space-between',
+        alignContent: 'space-between',
+      }}
+    >
+      {children}
+    </div>
+  )
+}
+
+const ExtraContainer: React.FC = ({ children }) => {
+  return (
+    <div
+      style={{
+        marginTop: '5px',
+      }}
+    >
+      {children}
+    </div>
+  )
+}
+
+function Header({ header }: { header: string }) {
+  const {
+    theme: { colors },
+  } = useContext(ThemeContext)
+  return (
+    <div style={smallHeaderStyle(colors)}>
+      <Trans>{header}</Trans>
```

Comment:
@wraeth-eth have we tested if this \<Trans> works here? i see in some places you're passing a `t` object as the header arg

Reply:
I dont think this would work - looks like a noob mistake on my part 3 months ago

Reply:
The right course of action is the t object being passed

Reply:
noooooooob